<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="text/html;charset=iso-8859-1"/>
<link type="text/css" rel="stylesheet" href="/benchmarkservice.css"/>
<script src="/scripts.js" type="text/javascript"></script>
<title>Benchmark Selection <DVAR name="tit"></title>
</head>
<body>
<div id="main">
<div id="mainLogo"></div>
<h2>Benchmark Selection for <DVAR name="tit"></h2>
Select from the available benchmarks and against which standard 
orthology inference project you would like to compare your predictions
<form action="gateway.pl" method="post">
 <DBOOL name="notRefSet">
 <fieldset><legend>Test Strategy</legend>
  <div class="notes"><h4>Select your strategy of testing</h4>
   <p>Performing the <b>pairwise tests</b> allows you to compare the quality of your predicted orthologs against each selected project independently. Comparisons across the different reference projects are not valid, as the set of proteins in general differs.</p>
   <p class="last">Tests on the <b>intersection of several projects</b> allow for a ranking among all the selected projects. Since all the tests are based on the set of common proteins, the power of the test is more and more reduced as the space of possible orthologs gets smaller and smaller the more projects are selected.</p>
  </div>
  <div class="test">
   <input type="radio" id="pPairwise" name="pTestStrategy" value="pairwise" checked="checked" /><label for="pPairwise">Pairwise Comparisons of each selected project against <DVAR name="tit"></label> <br/>
   <input type="radio" id="pIntersect" name="pTestStrategy" value="intersect" /><label for="pIntersect">Protein intersection set of selected projects and <DVAR name="tit"></label>
  </div>
 </fieldset>
 </DBOOL>
 <DBOOL name="isRefSet">
  <input type="hidden" name="pTestStrategy" value="<DVAR name="ref">" />
 </DBOOL>

 <fieldset><legend>Phylogeny-Based Definition Benchmarks</legend>
  <div class="notes"><h4>Phylogeny-Based Definition</h4>
   <p class="last">According to the phylogenetic definition, two homologous genes are orthologs if they diverged through a speciation event. Therefore, the phylogenetic tree of a set of orthologs (a set of genes in which any pair is orthologous) has by definition the same topology as the corresponding species tree.</p>
  </div>
   <div class="reference">
   <p class="last"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000262">Altenhoff and Dessimoz, 2009, <b>Phylogenetic and Functional Assessment of Orthologs Inference Projects and Methods</b> <i>PLoS Computational Biology</i>,5(1): e1000262</a></p>
  </div> 

  <div class="test_kind">
   <div class="test">
    <h4><input type="checkbox" id="pTTEuk1" name="pTTEuk1" <DVAR name="styleEuk">/><label for="pTTEuk1">Eukaryotic Species Tree Discordance Benchmark</label></h4>
    <DBOOL name="hasEuk">
    <div class="testParam">
     <label class="test" for="pTTEukPrj1">Projects to compare with:</label><br/>
     <div class="testParamList"><DVAR name="TTEukProjects"></div>
    </div>
    </DBOOL>
   </div>
   
   <div class="test">
    <h4><input type="checkbox" id="pTTFun1" name="pTTFun1" <DVAR name="styleFun">/><label for="pTTFun1">Fungi Species Tree Discordance Benchmark</label></h4>
    <DBOOL name="hasFun">
    <div class="testParam">
     <label class="test" for="pTTFunPrj1">Projects to compare with:</label>
     <div class="testParamList"><DVAR name="TTFunProjects"></div>
    </div>
    </DBOOL>
   </div>
   
   <div class="test">
    <h4><input type="checkbox" id="pTTBac1" name="pTTBac1" <DVAR name="styleBac">/><label for="pTTBac1">Bacteria Species Tree Discordance Benchmark</label></h4>
    <DBOOL name="hasBac">
    <div class="testParam">
     <label class="test" for="pTTBacPrj1">Projects to compare with:</label>
     <div class="testParamList"><DVAR name="TTBacProjects"></div>
    </div>
    </DBOOL>
   </div>
  </div>

  <div class="test_kind">
   <div class="test">
    <h4><input type="checkbox" id="pSTDLuca1" name="pSTDLuca1" <DVAR name="styleSTDLuca">/><label for="pSTDLuca1">Generalized Species Tree Discordance Benchmark based on all genomes in the <a href="http://swisstree.vital-it.ch/species_tree" target="_new">QfO Species Phylogeny</a></label></h4>
    <DBOOL name="hasSTDLuca">
    <div class="testParam">
     <label class="test" for="pSTDLucaPrj1">Projects to compare with:</label><br/>
     <div class="testParamList"><DVAR name="STDLucaProjects"></div>
    </div>
    </DBOOL>
   </div>

   <div class="test">
    <h4><input type="checkbox" id="pSTDEuk1" name="pSTDEuk1" <DVAR name="styleSTDEuk">/><label for="pSTDEuk1">Generalized Species Tree Discordance Benchmark on the eukaryotic genomes in the <a href="http://swisstree.vital-it.ch/species_tree" target="_new">QfO Species Phylogeny</a></label></h4>
    <DBOOL name="hasSTDEuk">
    <div class="testParam">
     <label class="test" for="pSTDEukPrj1">Projects to compare with:</label><br/>
     <div class="testParamList"><DVAR name="STDEukProjects"></div>
    </div>
    </DBOOL>
   </div>
   
   <div class="test">
    <h4><input type="checkbox" id="pSTDVer1" name="pSTDVer1" <DVAR name="styleSTDVer">/><label for="pSTDVer1">Generalized Species Tree Discordance Benchmark on the vertebrate genomes in the <a href="http://swisstree.vital-it.ch/species_tree" htarget="_new">QfO Species Phylogeny</a></label></h4>
    <DBOOL name="hasSTDVer">
    <div class="testParam">
     <label class="test" for="pSTDVerPrj1">Projects to compare with:</label><br/>
     <div class="testParamList"><DVAR name="STDVerProjects"></div>
    </div>
    </DBOOL>
   </div>

   <div class="test">
    <h4><input type="checkbox" id="pSTDFun1" name="pSTDFun1" <DVAR name="styleSTDFun">/><label for="pSTDFun1">Generalized Species Tree Discordance Benchmark on the fungal genomes in the <a href="http://swisstree.vital-it.ch/species_tree" target="_new">QfO Species Phylogeny</a></label></h4>
    <DBOOL name="hasSTDFun">
    <div class="testParam">
     <label class="test" for="pSTDFunPrj1">Projects to compare with:</label><br/>
     <div class="testParamList"><DVAR name="STDFunProjects"></div>
    </div>
    </DBOOL>
   </div>


  </div>


  <div class="test_kind">
   <div class="test">
  <div class="reference">
   <p class="last"><a href="http://dx.doi.org/10.1093/bib/bbr034">Brigitte Boeckmann, Marc Robinson-Rechavi, Ioannis Xenarios, Christophe Dessimoz, 2011 <b>Conceptual Framework and Pilot Study to Benchmark Phylogenomic Databases Based on Reference Gene Trees</b> <i>Briefings in Bioinformatics</i>, 12:5 (pp. 474-484)</a></p>
   <p class="last">A detailed description of our implementation is provided <a href="/doc/refGeneTrees.pdf">here</a>.</p>
  </div> 
    <h4><input type="checkbox" id="pRefPhylo1" name="pRefPhylo1" <DVAR name="styleRefPhylo"> /><label for="pRefPhylo1">Agreement with Reference Gene Phylogenies (contributed by Brigitte Boeckmann)</label></h4>
    <DBOOL name="hasRefPhylo">
    <div class="testParam">
     <label class="test" for="pRefPhyloPrj1">Projects to compare with:</label>
     <div class="testParamList"><DVAR name="RefPhyloProjects"></div>
     <label class="test" for="<DVAR name="RefPhyloProb1_id">">Reference Gene Phylogenies:</label>
     <div class="testParamList"><DVAR name="RefPhyloProblems"></div>
    </div>
    </DBOOL>
   </div>
  </div>
  
  <div class="test_kind_last">
   <div class="test">
  <div class="reference">
   <p class="last"><a href="http://dx.doi.org/10.1093/nar/gkj118">Heng Li <i>et. al</i>, 2006 <b>TreeFam: a curated database of phylogenetic trees of animal gene families</b> <i>Nucl. Acids Res.</i>, 34(suppl 1): D572-D580 484</a></p>
  </div> 
    <h4><input type="checkbox" id="pAutoPhylo1" name="pAutoPhylo1" <DVAR name="styleAutoPhylo"> /><label for="pAutoPhylo1">Agreement with Semi-automated Reference Phylogenies (contributed by Fabian Schreiber)</label></h4>
    <DBOOL name="hasAutoPhylo">
    <div class="testParam">
     <label class="test" for="pAutoPhyloPrj1">Projects to compare with:</label>
     <div class="testParamList"><DVAR name="AutoPhyloProjects"></div>
     <label class="test" for="<DVAR name="AutoPhyloProb1_id">">Reference Gene Phylogenies:</label>
     <div class="testParamList"><DVAR name="AutoPhyloProblems"></div>
    </div>
    </DBOOL>
   </div>
  </div>

 </fieldset>

<!-- Test commented out based on QfO discussion from beginning May 2013. 
 <fieldset><legend>Homology of Multi-Domain Proteins Test</legend>
  <div class="test_kind_last">
   <div class="test">
   <div class="reference">
    <p class="last">A description of our implementation of this test is provided <a href="/doc/homologyBenchmark.pdf">here</a>.</p>
  </div> 

    <h4><input type="checkbox" id="pHom1" name="pHom1" <DVAR name="styleHom"> /><label for="pHom1">Detection Accuracy of Multi-Domain Protein (contributed by Jacob Joseph and Dannie Durand)</label></h4>
    <DBOOL name="hasHom">
    <div class="testParam">
     <label class="test" for="pHomPrj1">Projects to compare with:</label>
     <div class="testParamList"><DVAR name="HomProjects"></div>
     <label class="test" for="pHomProbHumMus">Datasets:</label>
     <div class="testParamList"><DVAR name="HomProblems"></div>
    </div>
    </DBOOL>
   </div>
  </div>
 </fieldset>
-->
 <fieldset><legend>Function-Based Definition Benchmarks</legend>
  <div class="notes"><h4>Function-Based Definition</h4>
   <p class="last">One of the main application of orthology is the propagation of functional annotation, because orthologs tend to have a similar function.</p>
  </div>



  <div class="test_kind">
   <div class="test">
  <div class="reference">
   <p class="last"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000262">Altenhoff and Dessimoz, 2009, <b>Phylogenetic and Functional Assessment of Orthologs Inference Projects and Methods</b> <i>PLoS Computational Biology</i>,5(1): e1000262</a></p>
  </div> 
    <h4><input type="checkbox" id="pGO1" name="pGO1" <DVAR name="styleGO"> /><label for="pGO1">Gene Ontology Benchmark</label></h4>
    <DBOOL name="hasGO">
    <div class="testParam">
     <label class="test" for="pGOPrj1">Projects to compare with</label>
     <div class="testParamList"><DVAR name="GOprojects"></div>
   
     <fieldset><legend>Restricted to Evidence codes</legend>
      <p>
      <input type="checkbox"  id="pGOFilEXP" name="pGOFilEXP" checked="checked"/><label for="pGOFilEXP"><span title="Inferred from Experiment">EXP</span></label>
      <input type="checkbox"  id="pGOFilIDA" name="pGOFilIDA" checked="checked"/><label for="pGOFilIDA"><span title="Inferred from Direct Assay">IDA</span></label>
      <input type="checkbox"  id="pGOFilIPI" name="pGOFilIPI" checked="checked"/><label for="pGOFilIPI"><span title="Inferred from Physical Interaction">IPI</span></label>
      <input type="checkbox"  id="pGOFilIMP" name="pGOFilIMP" checked="checked"/><label for="pGOFilIMP"><span title="Inferred from Mutant Phenotype">IMP</span></label>
      <input type="checkbox"  id="pGOFilIGI" name="pGOFilIGI" checked="checked"/><label for="pGOFilIGI"><span title="Inferred from Genetic Interaction">IGI</span></label>
      <input type="checkbox"  id="pGOFilIEP" name="pGOFilIEP" checked="checked"/><label for="pGOFilIEP"><span title="Inferred from Expression Pattern">IEP</span></label>
      </p>
      <p>
      <input type="checkbox"  id="pGOFilISS" name="pGOFilISS" /><label for="pGOFilISS"><span title="Inferred from Sequence or Structural Similarity">ISS</span></label>
      <input type="checkbox"  id="pGOFilISO" name="pGOFilISO" /><label for="pGOFilISO"><span title="Inferred from Sequence Orthology">ISO</span></label>
      <input type="checkbox"  id="pGOFilISA" name="pGOFilISA" /><label for="pGOFilISA"><span title="Inferred from Sequence Alignment">ISA</span></label>
      <input type="checkbox"  id="pGOFilISM" name="pGOFilISM" /><label for="pGOFilISM"><span title="Inferred from Sequence Model">ISM</span></label>
      <input type="checkbox"  id="pGOFilIGC" name="pGOFilIGC" /><label for="pGOFilIGC"><span title="Inferred from Genomic Context">IGC</span></label>
      <input type="checkbox"  id="pGOFilIBA" name="pGOFilIBA" /><label for="pGOFilIBA"><span title="Inferred from Biological aspect of Ancestor">IBA</span></label>
      <input type="checkbox"  id="pGOFilIBD" name="pGOFilIBD" /><label for="pGOFilIBD"><span title="Inferred from Biological aspect of Descendant">IBD</span></label>
      <input type="checkbox"  id="pGOFilIKR" name="pGOFilIKR" /><label for="pGOFilIKR"><span title="Inferred from Key Residue">IKR</span></label>
      <input type="checkbox"  id="pGOFilIRD" name="pGOFilIRD" /><label for="pGOFilIRD"><span title="Inferred from Rapid Divergence">IRD</span></label>
      <input type="checkbox"  id="pGOFilIC"  name="pGOFilIC"  /><label for="pGOFilIC" ><span title="Inferred by Curator">IC </span></label>
      <input type="checkbox"  id="pGOFilRCA" name="pGOFilRCA" /><label for="pGOFilRCA"><span title="Inferred from Reviewed Computational Analysis">RCA</span></label>
      <input type="checkbox"  id="pGOFilTAS" name="pGOFilTAS" /><label for="pGOFilTAS"><span title="Traceable Author Statement">TAS</span></label>
      <input type="checkbox"  id="pGOFilNAS" name="pGOFilNAS" /><label for="pGOFilNAS"><span title="Non-traceable Author Statement">NAS</span></label>
      </p>
      <p>
      <input type="checkbox"  id="pGOFilIEA" name="pGOFilIEA" /><label for="pGOFilIEA"><span title="Inferred from electronic annotation">IEA</span></label>
      <input type="checkbox"  id="pGOFilNR"  name="pGOFilNR"  /><label for="pGOFilNR"><span title="No biological Data available">NR</span></label>
      </p>
     </fieldset>
     
     <fieldset><legend>Similarity Measure</legend>
      <input type="radio" id="pGOMLinA" name="pGOMeasure" value="avg Sim"/><label for="pGOMLinA">average Lin Similarity</label><br/>
      <input type="radio" id="pGOMLinM" name="pGOMeasure" value="max Sim"/><label for="pGOMLinM">maximum Lin Similarity</label><br/>
      <input type="radio" id="pGOMInfA" name="pGOMeasure" value="avg Info"/><label for="pGOMInfA">average information content</label><br/>
      <input type="radio" id="pGOMInfM" name="pGOMeasure" value="max Info"/><label for="pGOMInfM">maximum information content</label><br/>
      <input type="radio" id="pGOMSchA" name="pGOMeasure" value="avg Schlicker" checked="checked"/><label for="pGOMSchA">average Schlicker Similarity</label>
     </fieldset>
    </div>
    </DBOOL>
   </div>
  </div>
  
  <div class="test">
  <div class="reference">
   <p class="last"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000262">Altenhoff and Dessimoz, 2009, <b>Phylogenetic and Functional Assessment of Orthologs Inference Projects and Methods</b> <i>PLoS Computational Biology</i>,5(1): e1000262</a></p>
  </div> 
   <h4><input type="checkbox" id="pEC1" name="pEC1" class="inputCheckbox" <DVAR name="styleEC"> /><label for="pEC1">Enzyme Classification (EC) Benchmark</label></h4>
   <DBOOL name="hasEC">
   <div class="testParam">
    <label for="pECPrj1">Projects to compare with</label>
    <div class="testParamList"><DVAR name="ECprojects"></div>
    
    <fieldset><legend>Similarity Measure</legend>
     <input type="radio" id="pECMLinA" name="pECMeasure" value="avg Sim"/><label for="pECMLinA">average Lin Similarity</label><br/>
     <input type="radio" id="pECMLinM" name="pECMeasure" value="max Sim"/><label for="pECMLinM">maximum Lin Similarity</label><br/>
     <input type="radio" id="pECMInfA" name="pECMeasure" value="avg Info"/><label for="pECMInfA">average information content</label><br/>
     <input type="radio" id="pECMInfM" name="pECMeasure" value="max Info"/><label for="pECMInfM">maximum information content</label><br/>
     <input type="radio" id="pECMSchA" name="pECMeasure" value="avg Schlicker" checked="checked"/><label for="pECMSchA">average Schlicker Similarity</label>
    </fieldset>
   </div>
   </DBOOL>
  </div>
 </fieldset>

 <fieldset class="submit">
  <div>
   <input type="hidden" name="f" value="RunTests" />
   <input type="hidden" name="pPID" value="<DVAR name="PID">" />
   <input type="submit" value="Compute &raquo;" />
  </div>
 </fieldset>
</form>
<DVAR name="content">
</div>
</body>
</html>
